tidy3d/components/structure.py

```diff
@@ -678,5 +680,17 @@ class MeshOverrideStructure(AbstractStructure):
         "the last added structure of ``enforce=True``.",
     )
 
+    @pydantic.validator("geometry")
+    def _box_only(cls, val):
+        """Ensure this is a box."""
+        if isinstance(val, Geometry):
+            if not isinstance(val, Box):
+                log.warning(
+                    "Override structures should be 'Box' as of 'tidy3d' version 2.8. "
+                    f"Given type of '{type(val)}'. Converting to bounding box."
```

Comment:
How about `Given type of '{type(val)}, using '{type(val)}.bounding_box' instead.`? Just as an extra hint to the user that if they pass `geometry.bounding_box` in their override structure definition they can avoid the warning.

Reply:
ok

Reply:
changed